incubator/minecraft/templates/datadir-pvc.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.Persistence.Datadir.Enabled -}}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}-datadir
+  annotations:
+    volume.alpha.kubernetes.io/storage-class: {{ .Values.Persistence.StorageClass | quote }}
```

Comment:
The only scenario I can think of where you wouldn't want persistence is in testing. You can disable the PVC and use an emptyDir by setting `Values.Persistence.Datadir.Enabled` to false in the event that you don't.